curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5ef9b03c81a63668521804df.md

```diff
@@ -9,6 +9,12 @@ dashedName: step-46
 
 The `id` attribute is used to identify specific HTML elements. Each `id` attribute's value must be unique from all other `id` values for the entire page.
 
+Here is an example of an `input` element with an `id` attribute:
```

Comment:
this still shows up more on the forum then it should IMO
https://forum.freecodecamp.org/search?q=step%2046%20%20%23html-css%20cat%20photo%20app%20after%3A2024-03-01

especially with responses like these
https://forum.freecodecamp.org/t/learn-html-by-building-a-cat-photo-app-step-46/681365
https://forum.freecodecamp.org/t/learn-html-by-building-a-cat-photo-app-step-46/685022

campers should feel more confident about working with attributes by this point but are still struggling with how to create an id.
So either we need to keep this example or update the hint text to better direct them to the correct answer